src/v/pandaproxy/error.h

```diff
@@ -77,6 +77,7 @@ enum class reply_error_code : uint16_t {
     subject_not_deleted = 40405,
     subject_version_soft_deleted = 40406,
     subject_version_not_deleted = 40407,
+    serialization_error = 40801,
```

Comment:
This matches return code of other REST proxies, however a timeout error code (408) doesn't really make sense to me.  A 406 would.  Thoughts?

Reply:
It's all about compatibility.